src/main/java/me/ryanhamshire/GriefPrevention/FlatFileDataStore.java

```diff
@@ -320,12 +320,17 @@ else if (this.getSchemaVersion() == 0)
                         List<String> containerNames = Arrays.asList(line.split(";"));
                         containerNames = this.convertNameListToUUIDList(containerNames);
 
-                        //sixth line is list of players who can use buttons and switches
+                        //sixth line is list of players who can access creatures
+                        line = inStream.readLine();
```

Comment:
You didn't actually resolve the problems here by correcting the line number comments. This elevates existing access trust and converts existing permission trust into access trust. A new type of trust should be the last line. You cannot insert it into the middle without providing a converter for existing data.